lib/service/storage/esWrapper.js

```diff
@@ -160,12 +160,14 @@ class ESWrapper {
       return error;
     }
 
-    this.kuzzle.log.info({
```

Comment:
Actual change (I don't know why all the cluster files appear on this PR..)

Reply:
You should try to rebase your branch on latest 2-dev commit or merge 2-dev on your branch 🙂

Reply:
I already merged 2-dev :(